src/pages/messaging/index.mdx

```diff
@@ -23,39 +23,41 @@ This document gives you an overview of the LiveChat messaging protocol, explaini
 
-4. Messages are sent and delivered even when the recipient (both customer and agent) is **offline**.
+4. If you have the [Messaging Mode](#messaging-mode) enabled on your licence, the messages can be sent and delivered even when the recipient (either customer or agent) is **offline**.
 
-5. **Multiple agents** can participate in a single chat.
+5. Since each chat consists of multiple different threads, **multiple agents** can participate in a single chat.
 
 6. Every user can have **multiple concurrent chats**. Read how it applies to agents ([chatting as agent](#chatting-as-agent)) and customers ([chatting as customer](#chatting-as-customer)).
 
 7. The algorithm that decides how chats are distributed between agents is called **routing**. It's documented in the [Routing](#chat-routing) section.
 
+### Messaging Mode
+
+Once the Messaging Mode is enabled on your license, the chats become **continous**, which means once they're started, they remain open. Thanks to this feature, it's possible to receive and send messages to users when either an agent or the customer is offline. This way, the messaging becomes asynchronous and allows for more seamles communication. [Read more about the LiveChat Messaging Mode here](https://www.livechat.com/help/messaging-mode/).
+
 ### Chatting as agent
 
-The maximum number of concurrent chats an agent can participate in is 99. It can be configured in the LiveChat App by the license Owner or Admin.
+The maximum number of concurrent chats an agent can participate in is **99**. It can be configured in the LiveChat Agent App by the license Owner or Admin.
 
 ### Chatting as customer
 
-By default, a customer can only have one chat started, unless he chats with agents who use different licences. The licence Owner or Admin can change that default value, modifying the `routing.max_customer_chats_count` property. However, in order to support this functionality, you would need to build a custom Chat Widget. For now, the LiveChat Chat Widget doesn't support it. You can build a Chat Widget from the ground up using the [Customer SDK](../extending-ui/extending-chat-widget/customer-sdk/).
-
-Chats are continuous so customers can always preview their chats' history. Yet, with **multiple concurrent chats** customers can sort their chats out thematically, making sure they know what each chat is about.
+One customer can only have one chat started, unless he chats with agents who use different licences. If you have the [Messaging Mode](https://www.livechat.com/help/messaging-mode/) enabled on your licence, the chats are continuous, and customers can always preview their chats' history. With **multiple concurrent chats** customers can sort their chats out thematically, making sure they know what each chat is about. If you don't use the Messaging Mode option on your licence, your licence's Owner or Admin can disable the chat history in the Agent App settings.
```

Comment:
```suggestion
One customer can only have one chat started unless he or she chats with agents who use different licences. If you have the [Messaging Mode](https://www.livechat.com/help/messaging-mode/) enabled for your licence, the chats are continuous, and customers can always preview their chat history. With **multiple concurrent chats**, customers can sort out their chats thematically, making sure they know what each chat is about. If you don't use the Messaging Mode option, your licence's Owner or Admin can disable the chat history in the Agent App settings.
```

Reply:
1. First sentence - I don't think "they" would go well with this sentence since we started it with "one customer".
2. I tried to avoid "on license" as it seems to be a literal translation from Polish.
3. I can't find it anywhere, but I'm pretty sure I've seen the recommendation to replace "Agent App", "LiveChat Agent App" with "LiveChat App".

Reply:
1. Sure, I replaced this with a gender-neutral pronoun somewhat automatically :D 
2. 👍 
3. :O I believe that you might be right with this, maybe this is from our convo with Ola Kosicka about the naming conventions 🤔